keps/sig-scheduling/3094-pod-topology-spread-considering-taints/README.md

```diff
@@ -711,7 +778,8 @@ Pick one more of these and delete the rest.
 Describe the metrics themselves and the reasons why they weren't added (e.g., cost,
 implementation difficulties, etc.).
 -->
-N/A
+
+No.
```

Comment:
Based on question `How can an operator determine if the feature is in use by workloads?`, the answer should be yes. But kind of outside the scope of this KEP. kubernetes/kubernetes#110643

Reply:
Thanks for this advices.